src/RenderMethods/RenderFieldTemplate.tsx

```diff
@@ -9,35 +9,53 @@ import { JSONSchema7 } from 'json-schema';
 // It will be used to add drag and drop area later
 
 export default function RenderFieldTemplate({ id, schema, children }: FieldTemplateProps) {
-  const { schema: vernaSchema, setSchema, uiSchema, setUiSchema } = useVerna();
-  const canAddField = id.split('_').length === 3;
-  const canAddSection = id.split('_').length === 2;
+  const { schema: vernaSchema, setSchema, uiSchema, setUiSchema, selector } = useVerna();
   const isRoot = id === 'root';
-  const ownProperties = Object.keys(schema.properties || {}).length === 0;
+  const isSection = schema.type === 'object';
```

Comment:
Hmmm... I think we will have to manage data structure of type `object`. So we should not restrict this type to section.